kitchen.yml

```diff
@@ -50,7 +50,7 @@ platforms:
     driver_config:
       run_command: /usr/lib/systemd/systemd
       provision_command:
-        - yum install -y udev
+        - yum check-update && yum install -y udev
```

Comment:
OK, this is causing a breakage:

```
STDERR: The command '/bin/sh -c yum check-update && yum install -y udev' returned a non-zero code: 100
---- End output of docker -H unix:///var/run/docker.sock build -f Dockerfile-kitchen20190226-5221-j1u7qd . ----
Ran docker -H unix:///var/run/docker.sock build -f Dockerfile-kitchen20190226-5221-j1u7qd . returned 100] on default-fedora
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
The command "bundle exec kitchen verify ${INSTANCE}" exited with 20.
```

This will need to be resolved before continuing.